src/naksu/naksu.go

```diff
@@ -16,6 +16,8 @@ const lowDiskLimit = 50 * 1024 * 1024 // 50 Gb
 
 // URLTest is testing URL for checking network connection
 const URLTest = "http://static.abitti.fi/usbimg/qa/latest.txt"
+// URLTestTimeout is timeout (in seconds) for checking network connection
+const URLTestTimeout = 10
```

Comment:
Ten seconds might be a bit overkill. I'd try 2-4 seconds first.

Reply:
Changed to 4.